src/freakble/ble.py

```diff
@@ -20,9 +20,6 @@
 
 DEFAULT_TIMEOUT = 5.0
 NORDIC_UART_SERVICE_UUID = "6e400001-b5a3-f393-e0a9-e50e24dcca9e"
-NORDIC_UART_TX_CHARACTERISTIC = "6e400002-b5a3-f393-e0a9-e50e24dcca9e"
-NORDIC_UART_RX_CHARACTERISTIC = "6e400003-b5a3-f393-e0a9-e50e24dcca9e"
-
```

Comment:
Please could you leave this, i will use them to more error reporting and filtering. We want this characteristics anyway, a device that has a write char but it's not the one we want it's not OK.

Reply:
Nevermind, it's easier to search form them.